### ISSUE TITLE
Cant run

### ISSUE THREAD

**Issue by MisaGu** [glook/webpack4-react-boilerplate#9]
Node: v12.18.4
run: yarn install 

```node
[webpack-cli] TypeError: cli.isValidationError is not a function
    at Command.<anonymous> (C:\Users\User\Downloads\webpack-typescript-react-master\node_modules\@webpack-cli\serve\lib\index.js:96:25)
error Command failed with exit code 2.
```

**Comment by claudesortwell** [comment]
@MisaGu Not sure if you figured it out yet, but yarn add webpack-cli --dev should fix your issue

**Comment by vkvarma** [comment]
upgrade to latest version of library this will work :- ``` "dependencies": {
        "@types/classnames": "2.2.10",
        "@types/react": "16.9.53",
        "@types/react-dom": "16.9.8",
        "classnames": "2.2.6",
        "normalize.css": "8.0.1",
        "react": "17.0.1",
        "react-dom": "17.0.1"
    },
    "devDependencies": {
        "@babel/core": "7.12.3",
        "@babel/plugin-proposal-class-properties": "7.12.1",
        "@babel/plugin-proposal-export-default-from": "7.12.1",
        "@babel/plugin-proposal-export-namespace-from": "7.12.1",
        "@babel/plugin-proposal-object-rest-spread": "7.12.1",
        "@babel/plugin-proposal-throw-expressions": "7.12.1",
        "@babel/plugin-syntax-dynamic-import": "7.8.3",
        "@babel/plugin-transform-runtime": "7.12.1",
        "@babel/preset-env": "7.12.1",
        "@babel/preset-react": "7.12.1",
        "@babel/register": "7.12.1",
        "@pmmmwh/react-refresh-webpack-plugin": "0.4.2",
        "@svgr/webpack": "5.4.0",
        "@teamsupercell/typings-for-css-modules-loader": "2.4.0",
        "@typescript-eslint/eslint-plugin": "4.7.0",
        "@typescript-eslint/parser": "4.7.0",
        "autoprefixer": "10.0.1",
        "babel-eslint": "10.1.0",
        "babel-loader": "8.2.1",
        "clean-webpack-plugin": "3.0.0",
        "copy-webpack-plugin": "6.3.0",
        "core-js": "3.7.0",
        "css-loader": "5.0.0",
        "cssnano": "4.1.10",
        "eslint": "7.13.0",
        "eslint-config-airbnb-base": "14.2.0",
        "eslint-config-airbnb-typescript": "12.0.0",
        "eslint-config-prettier": "6.15.0",
        "eslint-import-resolver-alias": "1.1.2",
        "eslint-plugin-import": "2.22.1",
        "eslint-plugin-jsx-a11y": "6.4.1",
        "eslint-plugin-react": "7.21.5",
        "eslint-plugin-react-hooks": "4.2.0",
        "eslint-webpack-plugin": "2.2.1",
        "fork-ts-checker-webpack-plugin": "6.0.1",
        "html-loader": "1.3.2",
        "html-webpack-plugin": "5.0.0-alpha.6",
        "husky": "4.3.0",
        "import-sort-style-module-and-prefix": "0.1.3",
        "is-windows": "1.0.2",
        "less": "3.12.2",
        "less-loader": "7.0.2",
        "lint-staged": "10.5.0",
        "mini-css-extract-plugin": "1.3.0",
        "path": "0.12.7",
        "postcss-loader": "4.0.4",
        "prettier": "2.1.2",
        "prettier-plugin-import-sort": "0.0.6",
        "pretty-quick": "3.1.0",
        "react-refresh": "0.9.0",
        "regenerator-runtime": "0.13.7",
        "resolve-url-loader": "3.1.2",
        "sass": "1.29.0",
        "sass-loader": "10.0.4",
        "sass-resources-loader": "2.1.1",
        "style-loader": "2.0.0",
        "svg-url-loader": "6.0.0",
        "terser-webpack-plugin": "5.0.1",
        "ts-loader": "8.0.7",
        "typescript": "4.0.3",
        "url-loader": "4.1.1",
        "webpack": "5.21.2",
        "webpack-bundle-analyzer": "4.1.0",
        "webpack-cli": "4.5.0",
        "webpack-dev-server": "3.11.2",
        "webpack-merge": "5.7.3",
        "yargs": "16.1.0"
    },``` 
![image](https://user-images.githubusercontent.com/20686491/107606768-1d786d80-6c05-11eb-91af-bd8ff0d78fe4.png)

**Comment by soumyart** [comment]
@vkvarma still not working

![image](https://user-images.githubusercontent.com/20027561/109104375-dedcca00-7751-11eb-8d43-5102013b0025.png)

**Comment by claudesortwell** [comment]
@soumyart You don't have post css installed. Run `npm install postcss` or `yarn add postcss`

**Comment by glook** [comment]
@MisaGu Hi, thank you for your issue. You right, i fixed this issue in [v1.0.5](https://github.com/glook/webpack-typescript-react/releases/tag/v1.0.5).
